STYLE_GUIDE.md

```diff
@@ -40,79 +42,79 @@ Typical command structures are:
+The exception to the rule is if you want to write a bulk command, where the type of the arguments are the same and
 their order don’t matter, such as `ecs env delete test-pdx test-iad`.
 
 ### Flags
-Flags are used when you need more than one required input of different type or if the inputs are optional. 
-Every flag needs to have a description and should [use full sentences](#use-full-sentences). 
+Flags are used when you need more than one required input of different type or if the inputs are optional.
+Every flag needs to have a description and should [use full sentences](#use-full-sentences).
 All flag names are lowercase and the same across commands. Long flag names can use hyphens if necessary to disambiguate.
 
-Flags can also have a single character short name such as `env init -n MyEnv -p MyProject`. 
+Flags can also have a single character short name such as `env init -n MyEnv -p MyProject`.
 Required flags must have a short name, only provide a short name to an optional flag if you think it will be frequently used.
 
-Optional flags must have a default value. We want users to have the freedom to take complex operations, 
+Optional flags must have a default value. We want users to have the freedom to take complex operations,
 but by providing sane defaults we reduce the cognitive load to accomplish a task.
 
 ### Prompting
 Prompt the user for any required flag that the user did not provide a value for. Prompting is a human-friendly way of showing options:
 ![prompt](https://user-images.githubusercontent.com/879348/65549185-e4394380-ded1-11e9-9fce-f1b19cd4f27f.png)
 
-Don’t prompt the user if a flag value was provided to enable scripting.  
-Prompts should be written to `stderr`.  
-Highlight the important words in a prompt by bolding and italicizing the words. 
+Don’t prompt the user if a flag value was provided to enable scripting.
+Prompts should be written to `stderr`.
+Highlight the important words in a prompt by bolding and italicizing the words.
 
 ### Deleting and modifying production resources
-Any command that deletes or acts on a production resource should prompt the user to make sure they want to proceed with the action. 
+Any command that deletes or acts on a production resource should prompt the user to make sure they want to proceed with the action.
 The flag `-y, --yes` should be present in these commands to skip confirmation.
 
 ## Autocomplete
 Command names and flag names should be autocompleted using the <tab><tab> keys.
 
 ## Use full sentences
-Guidance text for the user must use full sentences, begin with a capital letter, and end with punctuation.   
-For example, the description for `env init` is “Deploy a new environment to your project.” and not “deploy a new environment to your project”. 
+Guidance text for the user must use full sentences, begin with a capital letter, and end with punctuation.
```

Comment:
Do we want to remove these extra spaces? I believe this will move "For example ..." to the next line, whereas now it wont.

Reply:
Oh hm, let me take a gander. My IDE did it, but lemme see what it looks like.